src/data/markdown/docs/01 guides/02 Using k6/14 Scenarios.md

```diff
@@ -55,726 +57,30 @@ export let options = {
 
 </div>
 
-### Common configuration
+## Common options
```

Comment:
This should still be an `h3` as it's part of the Configuration section.

Reply:
While you're correct, I think I'd like to keep it an h2 for now, as I want it to be displayed in the sidebar for smoother navigation. 🙇‍♂️